### PR TITLE
refactor(ekf_localizer): add Simple1DFilter params to parameter file

### PR DESCRIPTION
**Pull request by meliketanrikulu**
## Description
Simple1DFilter parameters will be added to the parameter file to be able to change these parameters from the parameter file.

This PR related this PR: https://github.com/autowarefoundation/autoware.universe/pull/5707

## Tests performed

Not applicable.

## Effects on system behavior

Not applicable.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Review by kminoda — COMMENTED** [review]
LGTM

**Comment by kminoda** [comment]
@meliketanrikulu Please fix pre-commit

File: autoware_launch/config/localization/ekf_localizer.param.yaml [hunk 1]
```diff
@@ -22,6 +22,11 @@
     proc_stddev_vx_c: 10.0
     proc_stddev_wz_c: 5.0
 
+    #Simple1DFilter parameters
+    z_filter_proc_dev: 1.0
+    roll_filter_proc_dev: 0.01
+    pitch_filter_proc_dev: 0.01
+
```
Comment (KYabuuchi) [inline autoware_launch/config/localization/ekf_localizer.param.yaml:29]: There are unnecessary whitespaces. It is causing pre-commit violations.  Please remove them.

```suggestion

```

About pre-commit: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/ci-checks/#pre-commit

**Review by KYabuuchi — CHANGES_REQUESTED** [review]

**Comment by KYabuuchi** [comment]
@kminoda This PR has long been left undone. This is not good so I have committed myself to it.
Please review and approve this :pray:

**Review by kminoda — APPROVED** [review]

**Review by KYabuuchi — APPROVED** [review]

**Pull request merged**